unit_testing/cli/test_cli_sct_image.py

```diff
@@ -12,3 +14,9 @@ def test_sct_image_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+
+@pytest.mark.parametrize("output_format", ('sct', 'fslhd', 'nibabel'))
+def test_sct_image_show_header_no_checks(output_format):
+    """Run the CLI script without checking results."""
+    sct_image.main(argv=['-i', 'sct_testing_data/t2/t2.nii.gz', '-header', output_format])
```

Comment:
> > I'm still not quite sure about this kind of test.
> > All that sct_image -header does is echo the values already stored in the image. So, if any problems were to arise, then the root cause is more likely to be because of the image itself (and not sct_image -header). Since most problems would be data-dependent, I'm not sure that a test using cached values would catch anything for us.
> 
> hum... yes, fair point. How about a simple functional test just to make sure the function doesn't crash?

:wink: